docs/source/contributing.rst

```diff
@@ -99,59 +99,67 @@ Contributing Code
 
 A feature development workflow might look like this:
 
+
 1. Follow the instructions in `Installing a Development Version`_.
 
+
 2. Sync with the upstream repository:
 
-::
+   ::
+
+      git pull --rebase origin main --recurse-submodules
 
-   git pull --rebase origin main --recurse-submodules
 
 3. Create a branch to develop from:
 
-::
+   ::
+
+      git checkout -b name-of-change
```

Comment:
Don't we have further guidelines on branch names somewhere?

Reply:
Yep! Thank you for spotting that, I will update it with the guideline.